### ISSUE TITLE
Cannot find module '../Modules/Staff/MuteCommand'

### ISSUE THREAD

**Issue by DylanSimowitz** [ewang20027/RaidBot#1]
Thanks for making this bot it's awesome! I set it up on my server last night and it's really impressive (especially the menu with reactions). I have found a couple of bugs and I plan on submitting merge requests, however right now I'm experiencing `Cannot find module '../Modules/Staff/MuteCommand'`

I think you just forgot to commit the particular file. Thanks for all your hard work again!

**Comment by ewang20027** [comment]
Hello @DylanSimowitz, 
Thank you for your interest in this project! This project is actually a WIP, so there will be a lot of issues. The MuteCommand was actually in both the Staff and Moderator folder; I already acknowledged this but haven't pushed the updates to Github yet.

Later on, I'll separate the master branch with a developer branch. Do feel free to point out any bugs, logic issues, poor coding practices (I tend to stay up past midnight working on this), etc.

Thanks again. If you have any questions, let me know.

**Comment by DylanSimowitz** [comment]
A 'stable' master branch would definitely be a good idea. I'll keep looking over the codebase and I'll write some documentation along the way. I'll also let you know what was confusing to set up from a users perspective. When the bot becomes more feature complete, help commands could be implemented. Would you ever consider a web UI for configuration or do you think that's a bit much for the level of configuration of this bot?

**Comment by ewang20027** [comment]
Hey,
Enjoy looking over the Configure Section command file & the Raid Handler file -- those two files have a lot of stuff that could probably be optimized.

I will implement a help command soon. I already have the implementation (since the structure of this bot is very similar to the structure of a different one). 

A web UI was definitely not on my list of things to do, as I don't have any experience working with web stuff. However, I do hope to learn! I feel like a web UI could simplify configuration for this bot in so many ways, as there will be more configuration commands. 

Just out of curiosity, seeing that you have found this project, I take it that you play Realm of the Mad God? If so, what's your in-game name and (if applicable) Discord tag? 

Thanks again!

**Comment by DylanSimowitz** [comment]
As strange as this sounds, I've never played rotmg. I have a coworker who plays and was asking about Discord bots for his guild server. I found this one after some searching and it seemed like the most promising for the features he wants.

**Comment by ewang20027** [comment]
Hello,
That's very interesting to hear. Sadly, there aren't many open-source raid management bots out there that can do this, so that's part of the reason why I made this bot.

The main reason why I'm making this bot is because I'm working with someone on a new Realm dungeon raiding server, which has huge potential. I'm trying to get this bot done by tomorrow -- or at the very latest, next week -- since I want to get lots of testing in. 

A web UI is more of a "nice to have" feature rather than a "must have" feature; thus, while I'd like to see something like that, it's not a huge thing at the moment. Later, when I am satisfied with the bot's core features, I plan on adding some more advanced features -- one major example is the ability for my bot to directly scrape information from RealmEye ([RealmEye](https://www.realmeye.com/) is a website that allows Realm of the Mad God players to lookup information about other players and guilds; manage trade offers and find potential offers on RealmEye trading; communicate with other players through the Wiki; and more). Currently, I'm using a pre-made API to get basic information about a player; however, I want to be able to get all possible information about a player, such as the basics (name, current rank/fame & characters, basically everything listed in `https://www.realmeye.com/player/<name>`), name history, rank history, guild history, fame history, graveyard data, and more. I plan on learning how to scrape websites later, but I don't really have the time at the moment. If you want to contribute, I suppose this is the best way to start -- that, and looking over the code and finding mistakes/optimizing. 

If I may ask, what is your coworker's RotMG name/Discord tag? 

Thanks again! And sorry for the wall of text.

**Comment by DylanSimowitz** [comment]
I think his ign is TehNubShow and his Discord is HamSandwich#0743. I see the unofficial API for realmeye you're using for the verification. There's a package https://github.com/jakcodex/realmeye-api which unfortunately hasn't been updated in a couple of years but I think it's a good place to start. I have a lot of experience writing web scrapers so I could take a crack at it. Is realmeye created by the makers of rotmg? How is realmeye able to get all of their data?

**Comment by ewang20027** [comment]
Hey,
Yeah, I believe the developer doesn't intend on continuing with his API. If you would like, please do! I would greatly appreciate it.

RealmEye is not created by the makers of Realm. I _think_ that RealmEye has a bunch of bots on each server that logs players that have joined the server and then updates it on the website. In other words, when a player joins the main lobby (Nexus), the RealmEye bots "see" the player that just joined and updates the website accordingly. 

I can give you some examples of profiles that you can try to scrape. [SamRiddelI](https://www.realmeye.com/player/SamRiddelI) looks like he has a completely public profile (aside from last seen location). [Opre](https://www.realmeye.com/player/opre) has a hidden graveyard. [LSDarko](https://www.realmeye.com/player/LSDarko) is basically inactive (characters aren't hidden but no characters). 

Basically, RealmEye gives RotMG players the ability to hide parts of their profile. For example, with Opre, if you try to access [his graveyard](https://www.realmeye.com/graveyard-of-player/opre), you get the error "The graveyard of Opre is hidden." Below is an attachment of the settings a RealmEye user can change. 
![image](https://user-images.githubusercontent.com/37031713/80332608-21ffa100-8800-11ea-9a8e-2932caeb792b.png)

If you have any questions, let me know. Also, a request -- if you happen to finish the scraper, could you also include documentation on how everything works? I'd like to use this as a learning opportunity.

**Comment by ewang20027** [comment]
I will be closing this issue for now. Please feel free to open a new one if you have any other questions. Thanks again!
